Provider/BreadcrumbProvider.php

```diff
@@ -108,30 +110,41 @@ private function createBreadcrumbsFromRoutes(RouteCollection $routes)
         /** @var BreadcrumbCollectionInterface $breadcrumbs */
         $breadcrumbs = new $this->collectionClass();
 
-        $currentRoute = $routes->get($this->currentRoute);
-        $currentRouteName = $this->currentRoute;
+        $route = $routes->get($this->currentRouteName);
+        if (!$route) {
+            // we did not find the route of this request. play it safe
+            return $breadcrumbs;
```

Comment:
this is strange but depending on custom routing systems it could happen. we don't want to fail in this case